src/pages/u/[userId].tsx

```diff
@@ -13,17 +13,20 @@ const Profile = () => {
   }
   return (
     <Main>
-      <div className="p-5">
-      <div>
-      {user.name}
+      <div className="card w-96 bg-base-100 shadow-xl">
+        <figure><img src={user.profile.photo} alt="Profile Pic" /></figure>
+        <div className="card-body">
+          <h2 className="card-title">{user.name}</h2>
+          <p>{user.profile.bio}</p>
```

Comment:
profile will be undefined (ts error), make sure you extend the type here: https://github.com/anggxyz/kernel-searchers/blob/852faadfb3514b858bacb63aba0148c63c99664e/src/hooks/useUserFromUserId.ts#L10